[E2] Getting error

Error:
```
[ERROR] lua/entities/gmod_wire_expression2/base/preprocessor.lua:504: attempt to index local 'portstbl' (a nil value)
  1. ConvertDescriptions - lua/entities/gmod_wire_expression2/base/preprocessor.lua:504
   2. Process - lua/entities/gmod_wire_expression2/base/preprocessor.lua:392
    3. Execute - lua/entities/gmod_wire_expression2/base/preprocessor.lua:28
     4. Include - lua/entities/gmod_wire_expression2/cl_init.lua:22
      5. Validate - lua/entities/gmod_wire_expression2/cl_init.lua:80
       6. Validate - lua/wire/client/text_editor/wire_expression2_editor.lua:1702
        7. OnMousePressed - lua/wire/client/text_editor/wire_expression2_editor.lua:902
         8. unknown - lua/wire/client/text_editor/issue_viewer.lua:151

```

Relevant code:
```golo
@name Remote Network Data Transfer System (RNDT) v 0.0.1
@inputs Radio:wirelink Radio:entity Activate:number
@outputs Recieve:number [One, Two, Three, Four, Five, Six, Seven]:number #[Radio inputs]#
@persist 
@strict



#[
    Description of the each channel.




]#

if(first() | dupefinished()){
    #include "lib/rndt"
    
    
    ReadyToRecieve = 1 # will send signals that it is going to interpret the given data.
    TranfserChannels = 7 #Amount of recieving channels that radio has 
    Self = entity()   

    
}

interval(100)


function number isRecieving(){
    local Recieved = 0
    
    for(I = 1, TranfserChannels){
        local Key = Radio[I:toString(), number]        
        
        if(Key > 0){
            Recieved = 1
        }    
    }
    
    return Recieved
    
}

if(Activate){

    Recieve = isRecieving()


}    



```

--- a/lua/entities/gmod_wire_expression2/cl_init.lua
+++ b/lua/entities/gmod_wire_expression2/cl_init.lua
@@ -57,7 +57,7 @@ function E2Lib.Validate(buffer)
 
 	-- invoke preprocessor
 	local status, directives, buffer, preprocessor = E2Lib.PreProcessor.Execute(buffer)
-	if not status then table.Add(errors, directives) end
+	if not status then table.Add(errors, directives) return errors end
 	---@cast directives PPDirectives
 	table.Add(warnings, preprocessor.warnings)
 

